migration/faqs.md

```diff
@@ -0,0 +1,61 @@
+
+#### Do I need to be a miner to register as a validator?
+
+No, you don't. Any type of user can register as a validator, regardless of whether they are a miner or not.
+
+#### Do I need to register as a validator to participate actively in the migration?
+
+No, registration as a validator is optional. Users can choose to actively participate either as a validator or as an observer. Both options are available but participation in the migration is not mandatory for any Nimiq user.
+
+#### Can I deposit more than the minimum required as a pre-registered validator?
+
+Yes. Any NIM amount exceeding the minimum deposit of 100 000 NIM will be considered stake, but only if it exceeds the minimum stake of 100 NIM. If the excess does not meet this minimum, it will be burned.
+
+#### Will my balance remain intact during the transition?
+
+Yes, your balance and transaction history will remain intact throughout the transition.
+
+#### I don’t want to participate in the transition, but I want to earn rewards. How can I do it?
+
+You can participate in the pre-staking phase by delegating NIM to one of the pre-registered validators. Simply select a validator from the [provided list]() and pre-stake your NIM to earn rewards post-transition.
+
+#### Can I stake more than the minimum deposit of 100 NIM?
+
+Yes, you can. Any amount above the required 100 NIM minimum deposit will be added as additional stake to the validator of your choice.
+
+#### Can I pre-stake to a validator anytime during the transition?
+
+No, pre-staking is only available during the pre-staking phase. Your NIM will be added to the pre-registered validator’s deposit, contributing to the required threshold of stake needed for migration readiness.
+
+#### What happens if the activation doesn’t collect 80% of readiness?
+
+If the activation phase fails to achieve the 80% stake readiness target, a new activation window will start at the block immediately following the candidate block. Each new activation window spans 1440 PoW blocks, equivalent to one day, and this process will continue until the required readiness threshold is reached. For more details, you can find more information [here](migration-technical-details#activation-phase).
+
+#### I pre-registered as a validator but forgot to run the activation tool/send the readiness transaction within the activation window. Can I recover my validator?
+
+Your validator will still be included in the validator list and you can access to it once you migrate. However, if you are selected to produce a block and fail to do so because you didn't launch the tool in time, you will face penalties for the block you skipped to produce. Once you realize the oversight, you can run the activation tool to migrate the state and history and build the genesis block, and reactivate your validator via transaction.
```

Comment:
```suggestion
Your validator will still be included in the validator list and you can access to it once you migrate. However, if you are selected to produce a block and fail to do so because you didn't launch the tool in time, you will face penalties for the block you skipped to produce. Once you realize the oversight, you can run the activation tool to migrate the state and history and build the genesis block, and reactivate your validator via sending a transaction to the migrated PoS chain.
```